2022-02-18 13:05:45,693 INFO    MainThread:15407 [wandb_setup.py:_flush():71] setting env: {}
2022-02-18 13:05:45,693 INFO    MainThread:15407 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-18 13:05:45,694 INFO    MainThread:15407 [wandb_init.py:_log_setup():371] Logging user logs to /home/simon/CDE_UBS/thesis/testing/wandb/run-20220218_130545-3mbmnjck/logs/debug.log
2022-02-18 13:05:45,694 INFO    MainThread:15407 [wandb_init.py:_log_setup():372] Logging internal logs to /home/simon/CDE_UBS/thesis/testing/wandb/run-20220218_130545-3mbmnjck/logs/debug-internal.log
2022-02-18 13:05:45,694 INFO    MainThread:15407 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f37dac3f4d0>
2022-02-18 13:05:45,694 INFO    MainThread:15407 [wandb_init.py:init():404] calling init triggers
2022-02-18 13:05:45,694 INFO    MainThread:15407 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-02-18 13:05:45,695 INFO    MainThread:15407 [wandb_init.py:init():460] starting backend
2022-02-18 13:05:45,695 INFO    MainThread:15407 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-18 13:05:45,759 INFO    MainThread:15407 [backend.py:ensure_launched():216] starting backend process...
2022-02-18 13:05:45,818 INFO    MainThread:15407 [backend.py:ensure_launched():222] started backend process with pid: 15456
2022-02-18 13:05:45,821 INFO    MainThread:15407 [wandb_init.py:init():469] backend started and connected
2022-02-18 13:05:45,889 INFO    MainThread:15407 [wandb_run.py:_label_probe_notebook():911] probe notebook
2022-02-18 13:05:45,894 INFO    MainThread:15407 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-02-18 13:05:45,895 INFO    MainThread:15407 [wandb_init.py:init():533] updated telemetry
2022-02-18 13:05:46,044 INFO    MainThread:15407 [wandb_init.py:init():563] communicating current version
2022-02-18 13:05:48,129 INFO    MainThread:15407 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-18 13:05:48,131 INFO    MainThread:15407 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-18 13:05:48,345 INFO    MainThread:15407 [wandb_init.py:init():606] starting run threads in backend
2022-02-18 13:05:53,356 INFO    MainThread:15407 [wandb_run.py:_console_start():1810] atexit reg
2022-02-18 13:05:53,356 INFO    MainThread:15407 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-02-18 13:05:53,357 INFO    MainThread:15407 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-02-18 13:05:53,358 INFO    MainThread:15407 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-18 13:05:53,358 INFO    MainThread:15407 [wandb_init.py:init():633] run started, returning control to user process
2022-02-18 13:06:03,163 INFO    MainThread:15407 [wandb_init.py:_pause_backend():284] pausing backend
2022-02-18 13:06:03,164 INFO    MainThread:15407 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-18 13:06:58,556 INFO    MainThread:15407 [wandb_init.py:_resume_backend():293] resuming backend
2022-02-18 13:06:58,565 INFO    MainThread:15407 [wandb_init.py:_pause_backend():284] pausing backend
2022-02-18 13:06:58,566 INFO    MainThread:15407 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-18 13:06:58,643 INFO    MainThread:15407 [wandb_init.py:_resume_backend():293] resuming backend
2022-02-18 13:06:58,652 INFO    MainThread:15407 [wandb_init.py:_pause_backend():284] pausing backend
2022-02-18 13:06:58,652 INFO    MainThread:15407 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-18 13:06:58,843 INFO    MainThread:15407 [wandb_init.py:_resume_backend():293] resuming backend
2022-02-18 13:06:58,858 INFO    MainThread:15407 [wandb_setup.py:_flush():71] setting env: {}
2022-02-18 13:06:58,859 INFO    MainThread:15407 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-18 13:06:58,859 INFO    MainThread:15407 [wandb_init.py:_log_setup():371] Logging user logs to /home/simon/CDE_UBS/thesis/testing/wandb/run-20220218_130658-3gnxt8ob/logs/debug.log
2022-02-18 13:06:58,859 INFO    MainThread:15407 [wandb_init.py:_log_setup():372] Logging internal logs to /home/simon/CDE_UBS/thesis/testing/wandb/run-20220218_130658-3gnxt8ob/logs/debug-internal.log
2022-02-18 13:06:58,859 INFO    MainThread:15407 [wandb_init.py:init():404] calling init triggers
2022-02-18 13:06:58,859 INFO    MainThread:15407 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-02-18 13:06:58,859 INFO    MainThread:15407 [wandb_init.py:init():426] re-initializing run, found existing run on stack: 3mbmnjck
2022-02-18 13:06:58,861 INFO    MainThread:15407 [wandb_run.py:finish():1457] finishing run simon-donike/SRCNN/3mbmnjck
2022-02-18 13:06:58,862 INFO    MainThread:15407 [jupyter.py:save_history():429] not saving jupyter history
2022-02-18 13:06:58,862 INFO    MainThread:15407 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-18 13:06:58,862 INFO    MainThread:15407 [wandb_init.py:_jupyter_teardown():303] cleaning up jupyter logic
2022-02-18 13:06:58,863 INFO    MainThread:15407 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-02-18 13:06:58,864 INFO    MainThread:15407 [wandb_run.py:_restore():1752] restore
2022-02-18 13:06:59,426 INFO    MainThread:15407 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 761
}

2022-02-18 13:06:59,671 INFO    MainThread:15407 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 761
}

2022-02-18 13:07:00,425 INFO    MainThread:15407 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 25181
}

2022-02-18 13:07:00,530 INFO    MainThread:15407 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 25181
}

2022-02-18 13:07:00,638 INFO    MainThread:15407 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 25181
}

2022-02-18 13:07:00,741 INFO    MainThread:15407 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25181
  total_bytes: 25181
}

2022-02-18 13:07:00,844 INFO    MainThread:15407 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25181
  total_bytes: 25181
}

2022-02-18 13:07:00,947 INFO    MainThread:15407 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25181
  total_bytes: 25181
}

2022-02-18 13:07:01,049 INFO    MainThread:15407 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25181
  total_bytes: 25181
}

2022-02-18 13:07:01,152 INFO    MainThread:15407 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25181
  total_bytes: 25181
}

2022-02-18 13:07:01,680 INFO    MainThread:15407 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25181
  total_bytes: 25181
}

2022-02-18 13:07:02,128 INFO    MainThread:15407 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25181
  total_bytes: 25181
}
local_info {
}

2022-02-18 13:07:04,042 INFO    MainThread:15407 [wandb_run.py:_append_files():2180] logging synced files
